client/blocks/importer/wordpress/import-everything/index.tsx

```diff
@@ -107,32 +108,46 @@ export class ImportEverything extends SectionMigrate {
 	};
 
 	recordMigrationStatusChange = ( prevState: State ) => {
+		const trackEventProps = {
+			sourceSiteId: this.props.sourceSiteId,
+			sourceSiteUrl: this.props.sourceUrlAnalyzedData?.url,
+			targetSiteId: this.props.targetSiteId,
+			targetSiteSlug: this.props.targetSiteSlug,
+			isTrial: isMigrationTrialSite( this.props.targetSite ),
```

Comment:
The key of event props needs to be snake case, otherwise it won't gets recorded correctly.

<img width="1276" alt="Screen Shot 2023-08-31 at 11 38 01 AM" src="https://github.com/Automattic/wp-calypso/assets/4074459/f7974c0a-99de-4c7d-aa4a-f461543b893f">